Moku/Customer/Extentions/DateExtention.swift

```diff
@@ -35,46 +43,46 @@ extension Date {
         if hari < 7 {
             let lastIndex = 7 + (7 - hari)
```

Comment:
What this is formula about? Is 7 magic number? what does it mean? Put 7 as constant with clear naming convention so easily manage in the future development